package.json

```diff
@@ -42,23 +42,28 @@
     "*.d.ts"
   ],
   "scripts": {
-    "chromatic": "chromatic -t 9b39ff142a7f",
     "build": "tsup",
     "build:staging": "CHROMATIC_BASE_URL=https://www.staging-chromatic.com tsup",
-    "build:watch": "pnpm run build --watch",
+    "build:watch": "run-p 'build --watch' 'codegen --watch'",
+    "build-storybook": "storybook build",
+    "chromatic": "chromatic -t 9b39ff142a7f",
+    "codegen": "graphql-codegen",
```

Comment:
Does this do typescript generation for fragments?

Reply:
I believe so. We use it in Chromatic as well, and we use fragments there.